tests/testsuite/build.rs

```diff
@@ -1462,6 +1521,7 @@ fn ignores_carriage_return_in_lockfile() {
     p.cargo("build").run();
 }
 
+#[allow(deprecated)]
 #[cargo_test]
 fn cargo_default_env_metadata_env_var() {
```

Comment:
Why wasn't this test updated?

Does seem like we need new literal redactions for this (if they don't cause problems)

Reply:
Maybe a regex that combines the two literals?

Reply:
Tend to leave it off for now. DLL prefix and suffix are not asserted much in other places.